client/init.c

```diff
@@ -70,9 +70,6 @@ int xdag_init(int argc, char **argv, int isGui)
 	if (!isGui) {
 		printf("%s client/server, version %s.\n", g_progname, XDAG_VERSION);
 	}
-	
-	/* initialize log system */
-	if (xdag_log_init()) return -1;
```

Comment:
I don't think it is good idea to move initialization of log system to the end of initialization

Reply:
It's original  here. I moved it to the beginning last time. And I found xdag_transport_start will fork a process from time to time then the logging system is not working.

Reply:
Ok, I see.